pkg/utils.py

```diff
@@ -291,7 +333,7 @@ def process_test(
     # merge the dataframe with all metrics
     if dataframe_list:
         merged_df = reduce(
-            lambda left, right: pd.merge(left, right, on="uuid", how="inner"),
+            lambda left, right: pd.merge(left, right, on="uuid", how="outer"),
```

Comment:
Why?

Reply:
From pandas [docs](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.merge.html)

> outer: use union of keys from both frames, similar to a SQL full outer join; sort keys lexicographically.
> inner: use intersection of keys from both frames, similar to a SQL inner join; preserve the order of the left keys.

Ok.

Reply:
Because some metrics that we are looking for might be present in a set of uuid runs and some in another. And when we do a join for those cases if uuids doesn't match we will get missed on a second set of uuids that have different metrics which are also interested in. So to make sure we get all the data related to our metrics of interest we need a full outer join.

Reply:
That makes sense. It helps with #122  to pull all data and at least print it in the csv. Even if it does not change the change point behavior.